src/main/java/edu/kis/powp/jobs2d/drivers/MacroDriver.java

```diff
@@ -0,0 +1,50 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import edu.kis.powp.jobs2d.features.DriverFeature;
+
+
+/**
+ * Driver command Manager.
+ */
+public class MacroDriver implements Job2dDriver {
```

Comment:
The name of the design pattern used should be added (in this case)

Reply:
How should this be written?